README.md

```diff
@@ -116,36 +98,40 @@ aws stepfunctions create-state-machine --endpoint http://localhost:8083 --defini
 
```

Comment:
I can't remember if you said something about this but the "Running the API locally with state machines + lambdas..." instruction starting on line 76 no longer works. I tried changing the name of the template but just got an error. Not sure if that part needs to come out or just be updated.